public/config/configuration.json

```diff
@@ -1,5 +1,5 @@
 {
-  "API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/businesses/",
+  "API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/businesses",
```

Comment:
I disagree with this. All our other URLs end with a slash and the danger with this one being different is that code may use this key, assume it has a slash and therefore not add one, and cause an error. I think it's better to be consistent within this project (and across projects).

Reply:
Please update where AUTH_URL is set in Actions.spec.ts -- it should end with a / just like the real configuration.

Reply:
Suggested solution for API_URL issue: change the URL to "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/" and then prepend "businesses" or "businesses/" when making the API calls.

Reply:
I like this suggestion, thanks!